runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -115,3 +110,192 @@ task_t<int64_t> f$openssl_verify(const string &data, const string &signature, co
 
   co_return 1;
 }
+
+static bool ichar_equals(char a, char b) noexcept {
+  return std::tolower(static_cast<unsigned char>(a)) == std::tolower(static_cast<unsigned char>(b));
+}
+
+static std::optional<tl::CipherAlgorithm> parse_algorithm(const string &method) noexcept {
```

Comment:
We prefer defining something in an anonymous namespace

Reply:
+